front_end/content.py

```diff
@@ -518,28 +556,6 @@ def problem_ids_to_titles(self):
             problem_dict[problem["problem_id"]] = problem["title"]
         return problem_dict
 
-    def get_student_courses(self, student_id):
-        courses = []
-        sql = '''SELECT courses.course_id, courses.title, submissions.user_id
-                 FROM courses
-                 INNER JOIN submissions ON courses.course_id = submissions.course_id
-                 WHERE submissions.user_id = ?'''
-        self.c.execute(sql, (str(student_id),))
-        for course in self.c.fetchall():
-            courses.append([course["title"], course["course_id"]])
-        return courses
-
-    def get_student_assignments(self, student_id):
-        assignments = []
-        sql = '''SELECT assignments.course_id, assignments.assignment_id, assignments.title
-                 FROM assignments
-                 INNER JOIN submissions ON assignments.assignment_id = submissions.assignment_id
-                 WHERE submissions.user_id = ?'''
-        self.c.execute(sql, (str(student_id),))
-        for assignment in self.c.fetchall():
-            assignments.append([assignment["title"], assignment["course_id"], assignment["assignment_id"]])
-        return assignments
-
     def get_student_problem_status(self, problem_id, student_id):
         passed = False
         sql = '''SELECT passed
```

Comment:
This function needs some more vertical white space.

Reply:
Generally, I like to have no more than 4-6 lines without any vertical white space. There are surely some exceptions to this, but having white space makes the code easier to read.